pkgs/top-level/python-packages.nix

```diff
@@ -4631,7 +4631,7 @@ in {
 
   matchpy = callPackage ../development/python-modules/matchpy { };
 
-  mathlibtools = callPackage ../development/python-modules/mathlibtools { };
+  mathlibtools = callPackage ../development/python-modules/mathlibtools { tqdm = callPackage ../development/python-modules/tqdm { usePandas = false; }; };
```

Comment:
```suggestion
  mathlibtools = callPackage ../development/python-modules/mathlibtools { tqdm = self.tqdm.override { usePandas = false; }; };
```

Reply:
This is only allowed in pythonPackages if the packages are only used in checkInputs. Otherwise this create really strange errors like empty derivations which pass all tests.

Reply:
I was only showing how we generally do those kinda of overrides, I was not going to approve the PR anyway thanks to the other issue.